ci/after_build_win32.bat

```diff
@@ -14,7 +14,7 @@ cd "build-win32/_CPack_Packages/win32/NSIS"
 
 set cronUpload=False
 
-if "%APPVEYOR_SCHEDULED_BUILD%" == "True"  if "%DUKTAPE_SUPPORT%" == "ON" set cronUpload=True
+if "%APPVEYOR_SCHEDULED_BUILD%" == "True"  if "%APPVEYOR_REPO_BRANCH%" == "master"  if "%DUKTAPE_SUPPORT%" == "ON" set cronUpload=True
```

Comment:
AppVeyor by default allows you to retrieve build artifacts (see https://www.appveyor.com/docs/status-badges/ for more info).

Example from my fork: https://ci.appveyor.com/project/dwrobel/pxCore/branch/onemw-wayland-egl-39/artifacts

Reply:
Hi Dworbel, 

Actually this upload is for providing pxscene edge builds to users. The requirement is to run build on daily basis and upload to a specific place. So that place can be referred in pxscene.org  for downloading.
So in this file, I am uploading pxscene setup to build server only when the cron job (APPVEYOR_SCHEDULED_BUILD) is true and the branch is master.